packs/equipment-effects/effect-rowan-rifle-cold.json

```
@@ -4,7 +4,7 @@
     "name": "Effect: Rowan Rifle (Cold)",
     "system": {
         "description": {
-            "value": "<p>The <em>Rowan Rifle</em>'s damage type changes from its current damage type to cold. On the next sunrise, the <em>Rowan Rifle</em> returns to dealing fire damage.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Rowan Rifle]</p>\n<p>The <em>Rowan Rifle</em>'s damage type changes from its current damage type to cold.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Rowan Rifle]</p>\n<p>The <em>Rowan Rifle's</em> damage type changes from its current damage type to cold.</p>"
```

I'd like to figure out a way to remove defunct effects, but this is still probably links in some person's game.